### PR TITLE
Jprouse/us148303 layouts fable testing

### PR DESCRIPTION
**Pull request by JakeProuse**
Rally Story: https://rally1.rallydev.com/#/?detail=/userstory/684998877825&fdp=true

Solving issues from our Fable Testing.

### CODE REVIEW

File: components/plugins/layouts.js [hunk 1]
```diff
@@ -155,9 +156,23 @@ export function addLayoutsPlugin() {
 
 			for (const layout of layouts) {
 				layout.contentEditable = false;
+				layout.setAttribute('tabindex', '-1');
 
 				for (const column of layout.children) {
 					column.contentEditable = true;
+					column.setAttribute('tabindex', '-1');
+					column.onkeydown = e => {
+						if (e.keyCode !== 9) return;
```
Comment (grant-cleary) [inline components/plugins/layouts.js:165]: Does shift-tab behave as expected in here?

Comment (JakeProuse) [reply components/plugins/layouts.js:165]: Should shift-tab bring you to the toolbar? Looks like I'm capturing that so I'll need to add a case to ignore it

Comment (JakeProuse) [reply components/plugins/layouts.js:165]: Ah similar to the original tab when I ignore the shift key, shift tab will bring you above the layout, then you'd need to shift-tab again to hit the toolbar.

Comment (JakeProuse) [reply components/plugins/layouts.js:164]: Also able to intercept that so it behaves as normal. I hate to code around things like this, it does make it kind of a pain if we added more shortcuts in the future.

Comment (grant-cleary) [reply components/plugins/layouts.js:164]: Yeah, hopefully we can find a better way to get this working in the future. Right now this is just the limitation of what we have available.

**Review by grant-cleary — APPROVED** [review]

**Pull request merged**